src/components/Modal/useShowHideModal.ts

```diff
@@ -1,8 +1,22 @@
-import { useEffect, useCallback } from "react";
+import { useCallback, useEffect } from "react";
 import useAnimationProps from "./useAnimationProps";
 import useKeyEvent from "../../hooks/useKeyEvent/index";
 
-export default function useShowHideModal({ instance, show, triggerElement, onClose, alertDialog }) {
+export default function useShowHideModal({
+  instance,
+  show,
+  triggerElement,
+  onClose,
+  alertDialog
+}: {
+  // the type is A11yDialog, but it's not exported, so can't be used
```

Comment:
lets create a `A11yDialogType` - have it set to any and once we have something better we could replace it in one place

Reply:
Done